### PR TITLE
Added default methods for sender, receiver, and identifier extraction to the MessageExtractionStrategy

### PR DESCRIPTION
**Pull request by ptorres-prowide**

### CODE REVIEW

**Comment by zubri** [comment]
Task linked: [CU-85ztvdjvp Agregar sender, receiver e identifier a MessageMetadataStrategy](https://app.clickup.com/t/85ztvdjvp)

File: src/main/java/com/prowidesoftware/swift/model/MessageMetadataStrategy.java [hunk 1]
```diff
@@ -48,4 +48,31 @@ public interface MessageMetadataStrategy {
      * Extracts the message trade date (only meaningful for some types of messages)
      */
     Optional<Calendar> tradeDate(AbstractMessage message);
+
+    /**
+     * Extracts the sender information from the message.
+     * This default implementation returns empty.
+     * @since 9.3.18
```
Comment (zubri) [inline src/main/java/com/prowidesoftware/swift/model/MessageMetadataStrategy.java:55]: version 19 en todos los casos

**Review by zubri — CHANGES_REQUESTED** [review]
@ptorres-prowide see review comment

File: src/main/java/com/prowidesoftware/swift/model/MessageMetadataStrategy.java (hunk 1 above)
Comment (ptorres-prowide) [reply src/main/java/com/prowidesoftware/swift/model/MessageMetadataStrategy.java]: ok

**Pull request merged**